wideq/client.py

```diff
@@ -138,6 +139,24 @@ def get_device(self, device_id) -> Optional['DeviceInfo']:
                 return device
         return None
 
+    def get_device_class(self, device_id):
```

Comment:
Aha, now that I see that you are not just getting the *class* but actually getting the device *object*, I think this probably needs a new name… maybe just `get_device_obj`? (I wish we had called the other one `get_device_info`, so we could call this just `get_device`, but I guess that's not worth changing now.)

Reply:
you're right, I should paid more attention to variables and function names :)